packages/app/src/components/nomination/nominationShow.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import NominationBanner from './nominationBanner'
+import nominationsAPI from '../../utils/API/nominationsAPI';
+
+const NominationShow = ({ match: { params: { id } } }) => {
+  const [NominationData, setNominationData] = useState({})
+  const [errorMessage, setErrorMessage] = useState(null)
+
+  useEffect(() => {
+    nominationsAPI.fetchNomination(id)
+      .then(function (response) {
+        const nomination = response.data.nomination
+        setNominationData(nomination)
+     })
+    .catch(function (err) {
+      setErrorMessage(err.response)
+    })
+  },[id])
```

Comment:
Nitpick: I'd expect there to be a space between the `,` and the `[`.

I also _think_ this line should end with a semicolon.